src/index.js

```diff
@@ -12,8 +12,12 @@ import Home from './js/home'
 import AuthPage from './js/authPage'
 import Question from './js/question'
 
-const NotFound = props => <h1>Sorry, could not find the page with URL <code>{props.url}</code></h1>
-
+const NotFound = props => <div>
```

Comment:
When you have multiple line components, I find it neater to have it the following way:
```js
const NotFound = props => (
  <div>
   ....
  </div>
)
```

Nevertheless that is just my preference